client/landing/stepper/declarative-flow/internals/steps-repository/sensei-plan/styles.scss

```diff
@@ -149,4 +177,13 @@
 			margin-bottom: 0;
 		}
 	}
+
+	.components-popover {
```

Comment:
For some reason, the popover was with the arrow white. Maybe some update broke that. These styles fix that.